thrimshim/thrimshim/main.py

```diff
@@ -138,15 +138,15 @@ def update_row(ident, new_row):
 	new_row['error'] = None
 
 	# actually update database
-	build_query = sql.SQL("""
+	query_str = """
 		UPDATE events
-		SET {}
+		SET {{}}
 		WHERE id = %(id)s
-		AND state IN ('UNEDITED', 'EDITED', 'CLAIMED')"""
-		).format(sql.SQL(", ").join(
-			sql.SQL("{} = {}").format(
-				sql.Identifier(column), sql.Placeholder(column),
-			) for column in new_row.keys()
+		{}""".format("AND state IN ('UNEDITED', 'EDITED', 'CLAIMED')" if not ('video_link' in new_row and new_row['video_link']) else "")
```

Comment:
rather than modifying the query directly, you can just do something like `AND state IN %(allowed_states)s`